src/test/java/net/sf/jabref/remote/DBConnectorTest.java

```diff
@@ -0,0 +1,58 @@
+package net.sf.jabref.remote;
+
+import java.sql.SQLException;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DBConnectorTest {
+
+    @Test
+    public void testGetNewMySQLConnection() {
+        try {
+            DBConnector.getNewConnection(DBType.MYSQL, "localhost", "jabref", "root", "");
```

Comment:
You should add a Assert here I think. Even though when it's just an assert that the connection is not null.